lib/dor/text_extraction/abbyy/file_watcher.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Dor
+  module TextExtraction
+    module Abbyy
+      # Watch jobs ABBYY is processing via the filesystem and report back to SDR
+      class FileWatcher
+        attr_reader :root_path, :result_xml_path, :exceptions_path
+
+        # These methods control the underlying listener; see:
+        # https://github.com/guard/listen?tab=readme-ov-file#pause--start--stop
+        # NOTE: You need to `sleep` or otherwise avoid letting the process
+        # exit after starting in order for the listener's thread to keep running
+        delegate :start, :pause, :stop, to: :listener
+
+        # Directories where ABBYY places OCR result information
+        RESULT_XML_DIR = 'RESULTXML'
+        EXCEPTIONS_DIR = 'EXCEPTIONS'
```

Comment:
These should be set in settings.

Reply:
Didn't realize we had that! Thanks

Reply:
Actually it's in windows. Might not be good.

Reply:
That might be OK if we set it to the windows version in production/on the VMs though, right? But we can leave it non-windows in local dev?

Reply:
Well it's only windows on the Abbyy vm, but the common-accessing vm mount is unix and at a different place.

Reply:
Ahh, and we use this setting for two different purposes? Maybe we need two settings then, or something...

Reply:
I split out the two path styles into separate values, and created https://github.com/sul-dlss/shared_configs/pull/2183.